packages/firebase_ui_auth/lib/src/views/phone_input_view.dart

```diff
@@ -121,7 +121,7 @@ class _PhoneInputViewState extends State<PhoneInputView> {
               widget.subtitleBuilder!(context),
             if (state is AwaitingPhoneNumber || state is SMSCodeRequested) ...[
               PhoneInput(
-                initialCountryCode: countryCode!,
+                initialCountryCode: countryCode,
```

Comment:
Will be most of the time null, because translations will mostly be on language code and not include a countryCode. It would be helpful to check `WidgetsBinding.instance.platformDispatcher.locale.countryCode` in case `Localizations.localeOf(context).countryCode` (L93) is null.